react_app/src/actions/auth.actions.js

```diff
@@ -33,7 +33,7 @@ export function login(formData) {
         .then(response => response.json().then(body => ({ body, response })))
         .then(({ body, response }) => {
           if (response.status === 200) {
-            dispatch(loginSuccess(response));
+            dispatch(loginSuccess(body));
```

Comment:
I think this was a typo in reference code, doesn't throw any errors but doesn't set state properly